README.Rmd

```diff
@@ -106,11 +106,9 @@ ccao::vars_dict %>%
     )
   ) %>%
   mutate(`Unique to Condo Model` = ifelse(
-    var_name_model != "loc_tax_municipality_name" & (
```

Comment:
We no longer need this conditional now that both repos are (temporarily) using `loc_cook_municipality_name`.